curriculum/challenges/english/25-front-end-development/quiz-error-handling/67f412eaf6e68343aab8ea81.md

```diff
@@ -17,439 +17,225 @@ To pass the quiz, you must correctly answer at least 18 of the 20 questions belo
 #### --text--
 
-Placeholder question
+What will be the output of this code?
 
-#### --distractors--
-
-Placeholder distractor 1
-
----
-
-Placeholder distractor 2
-
----
-
-Placeholder distractor 3
-
-#### --answer--
-
-Placeholder answer
-
-### --question--
-
-#### --text--
-
-Placeholder question
-
-#### --distractors--
-
-Placeholder distractor 1
-
----
-
-Placeholder distractor 2
-
----
-
-Placeholder distractor 3
-
-#### --answer--
-
-Placeholder answer
-
-### --question--
-
-#### --text--
-
-Placeholder question
-
-#### --distractors--
-
-Placeholder distractor 1
-
----
-
-Placeholder distractor 2
-
----
-
-Placeholder distractor 3
-
-#### --answer--
-
-Placeholder answer
-
-### --question--
-
-#### --text--
-
-Placeholder question
-
-#### --distractors--
-
-Placeholder distractor 1
-
----
-
-Placeholder distractor 2
-
----
-
-Placeholder distractor 3
-
-#### --answer--
-
-Placeholder answer
-
-### --question--
-
-#### --text--
-
-Placeholder question
-
-#### --distractors--
-
-Placeholder distractor 1
-
----
-
-Placeholder distractor 2
-
----
-
-Placeholder distractor 3
-
-#### --answer--
-
-Placeholder answer
-
-### --question--
-
-#### --text--
-
-Placeholder question
-
-#### --distractors--
-
-Placeholder distractor 1
-
----
-
-Placeholder distractor 2
-
----
-
-Placeholder distractor 3
-
-#### --answer--
-
-Placeholder answer
-
-### --question--
-
-#### --text--
-
-Placeholder question
+  ```py
+try:
+    print(22 / 0)
+except ZeroDivisionError:
+    print("You can't divide by zero!")
+  ```
 
 #### --distractors--
 
-Placeholder distractor 1
+22
 
 ---
 
-Placeholder distractor 2
+10
 
 ---
 
-Placeholder distractor 3
+0
 
 #### --answer--
 
-Placeholder answer
+You can't divide by zero!
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which module of the Python standard library lets you debug your code in an interactive way?
 
 #### --distractors--
 
-Placeholder distractor 1
+ABC
 
 ---
 
-Placeholder distractor 2
+numpy
 
 ---
 
-Placeholder distractor 3
+debugpy
 
 #### --answer--
 
-Placeholder answer
+pdb
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which exception does Python raise when you try to use a method or property that does not exist for that type?
 
 #### --distractors--
 
-Placeholder distractor 1
+`SyntaxError`
 
 ---
 
-Placeholder distractor 2
+`AttributeBug`
 
 ---
 
-Placeholder distractor 3
+`SyntaxBug`
 
 #### --answer--
 
-Placeholder answer
+`AttributeError`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which Python statement can you insert around various points in your code so you can see the values of variables while debugging?
 
 #### --distractors--
 
-Placeholder distractor 1
+`len()`
 
 ---
 
-Placeholder distractor 2
+`console()`
 
 ---
 
-Placeholder distractor 3
+`log()`
 
 #### --answer--
 
-Placeholder answer
+`print()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which module of the Python standard library lets you write unit tests?
```

Comment:
Unit tests are not included in these lectures. This one can be replaced with something about the `raise` statement, which has been recently added.